docs/iris/example_code/Meteorology/wind_speed.py

```diff
@@ -0,0 +1,69 @@
+"""
+Plotting wind direction using quiver
+===========================================================
+
+This example demonstrates using quiver to plot wind speed
+
+The second plot uses normalised data to produce arrows with a uniform size
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import iris
+import iris.coord_categorisation
+import iris.plot as iplt
+import iris.quickplot as qplt
+
+import cartopy
+
+def main():
+    # Load the u and v components of wind from a pp file
+    infile = iris.sample_data_path('wind_speed_lake_victoria.pp')
+
+    uwind = iris.load_cube(infile, 'x_wind')
+    vwind = iris.load_cube(infile, 'y_wind')
+
+    ulon = uwind.coord('longitude')
+    vlon = vwind.coord('longitude')
+    
+    # The longitude points go from 180 to 540, so subtract 360 from them
+    ulon.points = ulon.points - 360.0  
+    vlon.points = vlon.points - 360.0  
+
+    # Create a cube containing the wind speed
+    windspeed = ( uwind ** 2 + vwind ** 2 ) ** 0.5 
+    windspeed.rename( 'windspeed' )
+
+    x = ulon.points
+    y = uwind.coord('latitude').points
+    u = uwind.data
+    v = vwind.data
+
+    # Plot the wind speed as a contour plot
+    plt.figure()
```

Comment:
You don't actually need this line. Remember from the matplotlib course - if an axes doesn't have a current figure to be written onto then it will go away and create a new one :smiley: